### DISCUSSION TITLE
Collapsible submenu is different than what is shown in docs

### DISCUSSION THREAD

**Discussion by Vaibhav227** [saadeghi/daisyui#2105] [Q&A] [answered]
Why the chevron is in left?
![image](https://github.com/saadeghi/daisyui/assets/51129176/76aaafbf-61c8-42b0-8912-2414336cc10c)

**Comment by saadeghi** [comment, marked as answer]
I think this is daisyUI 2.x  
Install the latest version
```
npm i -D daisyui@latest
```

Also make sure your `content` config on `tailwind.config.js` file is addressing to all your template files and component files  
  Read more: https://tailwindcss.com/docs/content-configuration
